test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -880,6 +282,14 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
                 handler: 'index.handler',
                 disableLogs: true,
               },
+              functionWithVpc: {
```

Comment:
Let's follow same naming convention (name functions with `fn` prefix not `function` prefix)